### PR TITLE
Replaced domain name by hostname option while certificate creation

### PR DESCRIPTION
**Pull request by NimishaS**

### CODE REVIEW

File: lib/chef/knife/windows_cert_generate.rb [hunk 1]
```diff
@@ -28,15 +28,19 @@ class WindowsCertGenerate < Knife
...
+        then he must specify the hostname as: --hostname 'something.mydomain.com'. User can also give wildcard
+        as '*.mydomain.com'"
 
       option :output_file,
         :short => "-o PATH",
         :long => "--output-file PATH",
-        :description => "By default 3 files will be generated in the current folder as winrmcert.pfx and winrmcert.pem. You can specify alternate file path using this option. Eg: --output-file /home/.winrm/server_cert.pfx. This will create 3 files in the specified path with name server_cert.pem, sever_cert.pfx, server_cert.der.",
+        :description => "By default 3 files will be generated in the current folder as winrmcert.pfx and
+        winrmcert.pem. You can specify alternate file path using this option. Eg: --output-file /home/.winrm/server_cert.pfx.
+        This will create 3 files in the specified path with name server_cert.pem, sever_cert.pfx, server_cert.der.",
```
Comment (btm) [inline lib/chef/knife/windows_cert_generate.rb]: If we normally output three files, what happens when I provide ```--output-file foo.pfx```? Does it only generate the pfx file? Or does this just change the prefix of the file, e.g. ```--output-file /tmp/foo``` would generate three files with three different extensions with a filename of foo in /tmp?

There are also some inconsistencies in the description:

* the list of files is provided twice
* the first list of files says there are three files but only lists two
* We don't generate .der files anymore, instead we chose .b64

Comment (NimishaS) [inline lib/chef/knife/windows_cert_generate.rb]: We just verified that adding extension `pfx` in `output_file` is misleading. So `--output-file /home/.winrm/server_cert` will just create 3 files at path '/home/.winrm' with name 'server_cert'.
Updating the description accordingly.

**Comment by NimishaS** [comment]
@btm , can you please review this?

**Comment by adamedx** [comment]
:+1: , but please validate the end to end creation of a listener with a cert created this way.

**Comment by btm** [comment]
@NimishaS thanks :+1:

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @neshy

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

**Pull request merged**